stable/app/values.yaml

```diff
@@ -78,6 +78,8 @@ resources: {}
   # --   cpu: 100m
   # --   memory: 128Mi
 
+maxUnavailable : {}
```

Comment:
This should be 0 rather than a map since maxUnavailable is a value not a map.

Reply:
@punit-kulal this is updated, pls check